docs/UserGuide.md

```diff
@@ -56,11 +56,11 @@ Format: `help`
 
 Adds a patient to the patient list.
 
-Format: `addPatient n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS d/TASK_DESCRIPTION…`
+Format: `addPatient n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [d/TASK_DESCRIPTION]... [t/TAG]...`
```

Comment:
Would it be better if `…` is used rather than `...`? The former is less spaced apart and is used throughout the user guide, so it may be more consistent.